### DISCUSSION TITLE
Massive performance regression

### DISCUSSION THREAD

**Discussion by unphased** [nvim-treesitter/nvim-treesitter#5422] [General]
Today I see horrible insanely slow performance in my lua nvim config file when using commit b4f6dd72980607a9821d24502b0ca7ee826376af

I reverted nvim-treesitter back to commit 96a50241987531fdb8a516998b72c19f563ccd6c and it resolved the issue. 

Previously I had a similar issue with something broken that showed up in master: https://github.com/nvim-treesitter/nvim-treesitter/discussions/5247 In this discussion, I was told that `main` is the dev branch. But in every case Lazy is tracking `master` and `master` is the one that's broken for me.

Can you guys try to maintain stability better on master?

**Comment by unphased** [comment]
I have bisected the issue with lua performance regression down to the commit bfb4601f48c127ae6fb24a30ebf0495aaca8d28f in this repo.

**Reply by clason** [reply]
No, that's incorrect. The performance regression is from a Neovim master commit. Stick to Neovim 0.9.2 and you should be fine.
